packages/apps-config/src/api/spec/galois.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+export default {
+  Address: 'MultiAddress',
```

Comment:
There seems to be a couple of additional types missing, specifically -

```
REGISTRY: Unable to resolve type ExitReason, it will fail on construction
METADATA: Unknown types found, no types for ExitReason, Receipt, Transaction, TransactionStatus
```